src/__tests__/Panel.spec.js

```diff
@@ -5,11 +5,12 @@ import Adapter from 'enzyme-adapter-react-16';
 
 import Panel from '../components/Panel/Panel';
 
-Enzyme.configure({ adapter: new Adapter() });
-
 describe('<Panel />', () => {
   it('renders without crashing', () => {
-    const div = document.createElement('div');
-    ReactDOM.render(<Panel />, div);
+    shallow(<Panel />);
+  });
+  it('renders all Button Components', () => {
+    const wrapper = shallow(<Panel />);
+    expect(wrapper.find('Button').length).toBe(19);
```

Comment:
Good test, ensuring that all buttons necessary for the calculator's functionality are rendering to the panel! Nice.